crates/nu-command/tests/commands/every.rs

```diff
@@ -194,15 +134,7 @@ fn skips_every_third_row() {
             EmptyFile("tres.txt"),
         ]);
 
-        let actual = nu!(
-            cwd: dirs.test(), pipeline(
-            r#"
-                ls
-                | get name
-                | every 3 --skip
-                | to json --raw
-            "#
-        ));
+        let actual = nu!( cwd: dirs.test(), " ls | get name | every 3 --skip | to json --raw ");
```

Comment:
I have to say I find it harder to read when everything is crammed on to one line, especially when I try to see the difference between two test cases at a glance.

If it makes sense removing the multiline-raw-string the white-space around should possibly go as well?

Reply:
I agree. The intent of #8670 was to clean up tests where `pipeline()` is not doing anything useful; I'm OK leaving `pipeline()` calls in where they make the code more readable.

Reply:
I think the problem is not the removal of pipeline directly but rather the change away from the multiline strings. (might be that rustfmt starts to do its magic if it is removed)

Reply:
Yeah fair point. I was on the fence on cramming it in one line also. Will revert any obvious multi-line cases. Although, I'm not sure why the white-spaces around the quotes are still persisting even after I ran `cargo fmt --all`

Reply:
Thanks!

Reply:
Should be good to now I think, CI is passing too